.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     // some of these rules going forwards to clean up the code base.
     'import/no-extraneous-dependencies': 0,
     'import/no-unresolved': 0,
+    'import/extensions': 0,
```

Comment:
There is an extension for this eslint plugin that can make it webpack aware - we should add enabling this to our tech debt list, as it would allow checking that all imports (including aliased ones) are resolvable.